packages/lit-dev-content/site/tutorials/content/working-with-lists/01.md

```diff
@@ -0,0 +1,66 @@
+While Lit allows rendering values straight out of an iterable, it would be nice
+to have the ability to apply some transformation like adding markup for each
+item. Lit provides a built-in `map` directive that works on any iterable such as
+arrays, sets, maps, and even generators. It returns an iterable containing the
+result of calling the provided callback function on each item.
+
+In this example, a custom element is presented with a property named `items`
+that contains a set of strings.
+
+Render each item in the set as a list item (`<li>`) containing that string.
+Begin by importing the `map` directive.
+
+{% switchable-sample %}
+
+```ts
+// my-element.ts
+import {map} from 'lit/directives/map.js';
+```
+
+```js
+// my-element.js
+import {map} from 'lit/directives/map.js';
+```
+
+{% endswitchable-sample %}
+
+Use the directive in the rendered template, providing a callback function that
+returns a template wrapping each item with a list item (`<li></li>`) tag, and
+place it within the unordered list (`<ul></li>`) of the component's template.
```

Comment:
```suggestion
place it within the unordered list (`<ul></li>`) in the component's template.
```

Reply:
You could also just refer to the `<ul>` tag rather than "the unordered list":
```suggestion
place it inside the `<ul>` tag in the template.
```

Reply:
Oh, and `</li>` needs to be updated
```suggestion
place it within the unordered list (`<ul></ul>`) of the component's template.
```

Reply:
I like the shorter "`<ul>` tag".